mysql-test/suite/clone/include/clone_command.inc

```diff
@@ -0,0 +1,186 @@
+## Clone command test
+
+# These variables can to be set before sourcing this file. Currently we used
+# to test both local and remote clone.
+#
+# 1. Clone command is expected to return error
+# --let clone_err= <error number>
+#
+# 2. Test Remote Clone command. Default is local clone.
+# --let remote_clone = 1
+#
+# 3. Remote Clone command is expected to return error and the error number
+# is different from local clone.
+# --let clone_remote_err = <error number>
+#
+# 4. Skip clone_valid_donor_list configuration for testing error cases
+# --let skip_donor_config = 1
+#
+# 5. Test clone automatic tuning of threads
+# --let clone_auto_tune = 1
+#
+# 6. Test clone command forcing SSL [REQUIRES SSL]
+# --let clone_require_ssl = 1
+#
+# 7. Test clone command forcing insecure connection [REQUIRES NO SSL]
+# --let clone_require_no_ssl = 1
+#
+# 8. Test clone command forcing SSL certificate validation
+# --let clone_require_ssl_certificate = 1
```

Comment:
I am not sure about 4. and 5., but the SSL related options definitely do not seem to be applicable to the local clone. I would suggest to trim this file to the minimum.